baggers.js

```diff
@@ -13899,6 +13899,45 @@ var data = {
             "Paris",
             "Lyon"
         ]
+    },
+    {
+      "since": "2016-11-10",
+      "name": "Jean-Baptiste Beuzelin",
+      "bio": "Ingénieur R&D chez Lucca, je m'occupe de développer une solution interne de gestion de parc client et de facturation.",
+      "picture": "http://www.lucca.fr/img/equipe/jbeuzelin.png",
+      "websites": [
+        {
+          "name": "Lucca",
```

Comment:
Please use one of the name described in doc: https://github.com/brownbaglunch/bblfr_data#websites. I suggest here: `Web`